lib/zuora/objects/amendment.rb

```diff
@@ -6,14 +6,14 @@ class Amendment < Base
     validates_length_of :name, :maximum => 100
     validates_inclusion_of :auto_renew, :in => [true, false], :allow_nil => true
     validates_length_of :code, :maximum => 50, :allow_nil => true
-    validates_datetime_of :contract_effective_date, :allow_nil => true
-    validates_datetime_of :customer_acceptance_date, :allow_nil => true
-    validates_datetime_of :effective_date, :allow_nil => true
-    validates_datetime_of :service_activation_date, :if => Proc.new { |a| a.status == 'PendingAcceptance' }
+    # validates_datetime_of :contract_effective_date, :allow_nil => true
```

Comment:
Should these be commented or removed?

Reply:
i could be convinced to remove them if you thought it was worth it. i left them in because that's what upstream did and I was trying to get out of the merge conflict as cleanly as possible/leave visual clues for what changed because QAing this from core is a bit of a mystery